Mathlib/Tactic/NormNum/Pow.lean

```diff
@@ -237,7 +238,7 @@ blocks below were not necessary: we just did it once outside the `match rb with`
 -/
 /-- The `norm_num` extension which identifies expressions of the form `a ^ b`,
 such that `norm_num` successfully recognises both `a` and `b`, with `b : ℤ`. -/
-@[norm_num (_ : α) ^ (_ : ℤ)]
```

Comment:
I fixed this for you; the type annotation was just nonsense.

Reply:
Thanks a lot!